save-cosv/src/main/kotlin/com/saveourtool/save/cosv/storage/RawCosvFileS3KeyManager.kt

```diff
@@ -107,11 +107,15 @@ class RawCosvFileS3KeyManager(
     }
 
     @Transactional
-    override fun updateKeyByContentLength(key: RawCosvFileDto, contentLength: Long): RawCosvFileDto =
-            key.contentLength?.let {
-                repository.getByIdOrNotFound(key.requiredId())
-                    .let { entity ->
-                        repository.save(entity.apply { this.contentLength = contentLength }).toDto()
-                    }
-            } ?: key
+    override fun updateKeyByContentLength(
+        key: RawCosvFileDto,
+        contentLength: Long,
+    ): RawCosvFileDto = key.contentLength
+        ?.let { key }
```

Comment:
the main error

Reply:
Got it, actually I would have made the same mistake